FEXCore/Source/Interface/Core/Interpreter/Fallbacks/InterpreterFallbacks.cpp

```diff
@@ -279,6 +293,44 @@ bool InterpreterOps::GetFallbackHandler(const IR::IROp_Header* IROp, FallbackInf
   case IR::OP_VPCMPESTRX: *Info = {FABI_I32_I64_I64_V128_V128_I16, Core::OPINDEX_VPCMPESTRX}; return true;
   case IR::OP_VPCMPISTRX: *Info = {FABI_I32_V128_V128_I16, Core::OPINDEX_VPCMPISTRX}; return true;
 
+  // Float to GPR conversion fallbacks
+  case IR::OP_FLOAT_TOGPR_S: {
```

Comment:
I think you can drop all of these? They can be solved on the dispatcher side for f64 as CVT2SSI etc does rn, and f80 uses _F80CVTInt

Reply:
Thanks.